src/blocks/remote-data-container/index.ts

```diff
@@ -52,11 +56,109 @@ addFilter(
  */
 addFilter( 'blocks.registerBlockType', 'remote-data-blocks/addUsesContext', addUsesContext, 10 );
 
+interface State {}
+
+type Actions = {
+	GET_DATA: () => void;
+};
+
+interface Selectors {}
+
+const queryDataStateKey = (queryKey: string, blockName: string, queryInputs: Record<string, string>) => `${blockName}:${queryKey}:${JSON.stringify(queryInputs ?? {})}`;
+
+const remoteDataBlocksStoreConfig: ReduxStoreConfig< State, Actions, Selectors > = {
+	reducer: ( state = {}, action ) => {
+		switch ( action.type ) {
+			case 'RECEIVE_REMOTE_DATA':
+				// console.log('store: RECEIVED REMOTE DATA')
+				// console.log({action});
+				// console.log(`key: ${queryDataStateKey(action.queryKey, action.blockName, action.queryInput)}`);
+				return { ...state, [ queryDataStateKey(action.queryKey, action.blockName, action.queryInputs) ]: action.data };
```

Comment:
Each query response contains a UUID in the `resultId` property. Maybe we should use that?

Reply:
Nevermind, that won't work since the UUIDs will be different on subsequent page loads. 👎